docs: Added metadata to Economics (#1132)
starknet-io/starknet-docs: components/Starknet/modules/architecture_and_concepts/pages/Economics-of-Starknet.adoc

[id="economics_of_starknet"]
= The Economics of Starknet
:description: How economics work in a block chain in general, and specifically in Starknet. The purpose of the Starknet token, it's supply and distribution.
:keywords: tokenomics, STRK

> Starknet is a developing decentralized protocol and the economic mechanisms described here are subject to change based on governance decisions made by the larger community of Starknet. For more details on Starknet’s governance processes see the link:https://governance.starknet.io/[Starknet Governance Hub]. The technical documents provided here describe certain economic fundamentals of the Starknet token. These materials are intended for informational purposes only and are meant to outline the usage and functionalities of the asset within Starknet. It is important to understand that the primary purpose of the Starknet token is to facilitate operations and activities on Starknet and it is not intended to serve as an investment.

Author: ```suggestion
Starknet is a developing decentralized protocol and the economic mechanisms described here, also know as tokenomics, are subject to change based on governance decisions made by the larger community of Starknet. For more details on Starknet’s governance processes see the link:https://governance.starknet.io/[Starknet Governance Hub]. The technical documents provided here describe certain economic fundamentals of the Starknet token. These materials are intended for informational purposes only and are meant to outline the usage and functionalities of the asset within Starknet. It is important to understand that the primary purpose of the Starknet token is to facilitate operations and activities on Starknet and it is not intended to serve as an investment.
```

[id="economics_of_starknet"]

> = The Economics of Starknet

Author: ```suggestion
= The token economics of Starknet
```

> :description: How economics work in a block chain in general, and specifically in Starknet. The purpose of the Starknet token, it's supply and distribution.

Author: ```suggestion
:description: How token economics work in a block chain in general, and specifically in Starknet. The purpose of the Starknet token, its supply, and distribution.
```

> :keywords: tokenomics, STRK

Author: ```suggestion
:keywords: tokenomics, STRK, token economics, STRK
```

> Starknet is a developing decentralized protocol and the economic mechanisms described here, also know as tokenomics, are subject to change based on governance decisions made by the larger community of Starknet. For more details on Starknet’s governance processes see the link:https://governance.starknet.io/[Starknet Governance Hub]. The technical documents provided here describe certain economic fundamentals of the Starknet token. These materials are intended for informational purposes only and are meant to outline the usage and functionalities of the asset within Starknet. It is important to understand that the primary purpose of the Starknet token is to facilitate operations and activities on Starknet and it is not intended to serve as an investment.

Author: ```suggestion
#Starknet is a developing decentralized protocol and the economic mechanisms described here, also know as tokenomics, are subject to change based on governance decisions made by the larger community of Starknet. For more details on Starknet’s governance processes see the link:https://governance.starknet.io/[Starknet Governance Hub]. This document describes certain economic fundamentals of the Starknet token. This document is intended for informational purposes only and is meant to outline the usage and functionalities of the asset within Starknet. It is important to understand that the primary purpose of the Starknet token, STRK, is to facilitate operations and activities on Starknet and it is not intended to serve as an investment.#
```
